packages/styles/src/utilities/_variables.scss

```diff
@@ -28,6 +28,15 @@ $position-values: (
   100: 100%,
 );
 
+$font-family-sans-serif: var(--post-device-font-family-default);
```

Comment:
Device layer tokens (`post-device-*`) are not meant to be used directly, it seems like you are missing a utility token here. You can ask @Vandapanda to create it, it would be nice to have tokens for the font weights and line heights as well. They might be useful for the tailwind output anyways.